src/snowflake/cli/plugins/spcs/__init__.py

```diff
@@ -11,7 +11,7 @@
 
 app = SnowTyper(
     name="spcs",
-    help="Manages Snowpark services, pools, jobs, image registries, and image repositories.",
+    help="Manages Snowpark services, pools, image registries, and image repositories.",
```

Comment:
nit: Snowpark Container Services services, compute pools, image registries, and image repositories

Reply:
Noted, in the interest of keeping this PR focused, I'll make another PR to rename all other instances of Snowpark to Snowpark Container Services